CODEOWNERS

```diff
@@ -13,7 +13,7 @@
 /.github/workflows/ @cilium/github-sec @cilium/ci-structure @cilium/build
 /CODEOWNERS @cilium/contributing
 /images/bpftool @cilium/loader
-/images/checkpatch @cilium/bpf
+/images/checkpatch @cilium/sig-datapath
```

Comment:
GitHub is still complaining that this team doesn't exist :thinking:

Reply:
I think this is a permission problem that needs to be addressed in the repository's settings. /cc @aanm

Reply:
Seems fine now.